var/spack/repos/builtin/packages/py-graphql-relay/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyGraphqlRelay(PythonPackage):
+    """Relay library for graphql-core."""
+
+    homepage = "https://github.com/graphql-python/graphql-relay-py"
+    pypi = "graphql-relay/graphql-relay-3.2.0.tar.gz"
+
+    maintainers("LydDeb")
+
+    version("2.0.1", sha256="870b6b5304123a38a0b215a79eace021acce5a466bf40cd39fa18cb8528afabb")
+
+    depends_on("python@3.6:3.10", type=("build", "run"))
+    depends_on("py-setuptools@59:69", type="build")
+    depends_on("py-poetry-core@1", type="build")
```

Comment:
I don't see these for 2.0.1

Reply:
Absolutely, these are dependencies of version 3.2.0.
In the  pyproject.toml of the version 3.2.0 there are:
```
[build-system]
requires = ["poetry_core>=1,<2", "setuptools>=59,<70"]
```
Regarding version 2.0.1, there are only “setuptools” for the build backend but no version constraints. I'm trying to determine the maximum version of 'setuptools' for version 2.0.1